srcpkgs/gomi/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'gomi'
+pkgname=gomi
+version=1.1.1
+revision=1
+build_style=go
+go_import_path="github.com/b4b4r07/gomi/"
+short_desc="Replacement for UNIX rm command"
+maintainer="juan <juan@horlux.org>"
+license="MIT"
+homepage="https://babarot.me/gomi"
+changelog="https://github.com/b4b4r07/gomi/releases"
+distfiles="https://github.com/b4b4r07/gomi/archive/v${version}.tar.gz https://b4b4r07.mit-license.org/license.txt"
```

Comment:
the license is not in the distfile?

Reply:
@xtraeme  it is: https://b4b4r07.mit-license.org/license.txt it was just not provided within their repo directly but through that link in the readme.md

Reply:
It won't build with recent changes, [create_wrksrc](https://github.com/void-linux/void-packages/blob/master/Manual.md#optional-variables)=yes is needed.